cookbook/request-user-for-authentication/README.md

```diff
@@ -0,0 +1,74 @@
+# Requesting a user to authenticate
+
+When your app requires authentication, you should invite the user to login when he tries to access your app features.
+The SignIn app provides a signin form ready to be invoked from other Starcounter apps thanks to the blending approach.
+Here is the pattern for requesting authentication from your app:
+
+When your app decides that the view cannot be presented without authentication, it redirects to its own "Unauthenticated" view. 
+For exemple: `/your-app/partial/unauthenticated?return_uri={?}`
+
+<div class="code-name">MainHandlers.cs</div>
+
+```cs
+	if (!IsSignedIn()) {
+		return Self.GET("/your-app/partial/Unauthenticated?return_uri=" + getURL);
+	}
+```
+
+The `getURL` is the Url requested in the original GET request.
```

Comment:
this should be URI, not URL.

URL is a complete thing that starts from `http(s)://`